dist/constants.json

```diff
@@ -5883,931 +5883,931 @@
+        "label": "Shakopee VA Clinic"
       },
       {
         "value": "618GN",
-        "label": "LYLE C PEARSON CBOC"
+        "label": "Lyle C. Pearson Community Based Outpatient Clinic"
       },
       {
         "value": "618QB",
-        "label": "ELY VA CLINIC"
+        "label": "Ely VA Clinic"
       },
       {
         "value": "618",
-        "label": "MINNEAPOLIS VA HCS"
+        "label": "Minneapolis VA Medical Center"
       },
       {
         "value": "656",
-        "label": "ST. CLOUD VA HEALTH CARE SYSTEM"
+        "label": "St. Cloud VA Medical Center"
       }
     ],
     "IA": [
       {
         "value": "438GA",
-        "label": "SPIRIT LAKE CBOC"
+        "label": "Spirit Lake VA Clinic"
       },
       {
         "value": "636A8",
-        "label": "IOWA CITY HCS"
+        "label": "Iowa City VA Medical Center"
       },
       {
         "value": "636A6",
-        "label": "VA CIHS, DES MOINES DIVISION"
+        "label": "Des Moines VA Medical Center"
       },
       {
         "value": "636GP",
-        "label": "SHENANDOAH CBOC"
+        "label": "Shenandoah VA Clinic"
       },
       {
         "value": "636GX",
-        "label": "FORT DODGE NORTH VA CLINIC"
+        "label": "Fort Dodge North VA Clinic"
       }
     ],
     "SD": [
       {
         "value": "438GC",
-        "label": "SIOUX CITY CBOC"
+        "label": "Sioux City VA Clinic"
       },
       {
         "value": "438GD",
-        "label": "ABERDEEN CBOC"
+        "label": "Aberdeen VA Clinic"
       },
       {
         "value": "568A4",
-        "label": "HOT SPRINGS, SD MC"
+        "label": "Hot Springs VA Medical Center"
       },
       {
         "value": "568GA",
-        "label": "RAPID CITY CBOC"
+        "label": "Rapid City VA Clinic"
       },
       {
         "value": "568GB",
-        "label": "PIERRE CBOC"
+        "label": "Pierre VA Clinic"
       },
       {
         "value": "568HF",
-        "label": "PINE RIDGE"
+        "label": "Pine Ridge VA Clinic"
       },
       {
         "value": "568HG",
         "label": "SIOUX SAN"
       },
       {
         "value": "568HJ",
-        "label": "MISSION CBOC"
+        "label": "Mission VA Clinic"
       },
       {
         "value": "568HK",
-        "label": "MCLAUGHLIN VA CLINIC"
+        "label": "McLaughlin VA Clinic"
       },
       {
         "value": "568HP",
-        "label": "WINNER"
+        "label": "Winner VA Clinic"
       },
       {
         "value": "438GE",
-        "label": "WAGNER CBOC"
+        "label": "Wagner VA Clinic"
       },
       {
         "value": "438GF",
-        "label": "WATERTOWN CBOC"
+        "label": "Watertown VA Clinic"
       },
       {
         "value": "568",
-        "label": "BLACK HILLS HEALTH CARE SYSTEM - FT. MEADE DIVISION"
+        "label": "Fort Meade VA Medical Center"
       },
       {
         "value": "438",
-        "label": "SIOUX FALLS VA HCS"
+        "label": "Sioux Falls Regional Benefit Office"
```

Comment:
The new label for this facility code is Royal C. Johnson Veterans' Memorial Hospital, but this is going from a Health Care System to a single facility. So let's hold off until we figure out how to resolve that.